assets/sass/components/key-metrics/_googlesitekit-km-selection-panel.scss

```diff
@@ -101,6 +105,12 @@
 		flex-wrap: wrap;
 		justify-content: space-between;
 		margin-top: $grid-gap-desktop / 2;
+
+		@media (max-width: $bp-xsmallOnly) {
+			flex-direction: column;
+			margin-top: 0;
+			row-gap: 6px;
+		}
```

Comment:
How about using a mobile-first approach here as well, by only using `display: flex` for viewports above `$bp-tablet`? That way, the elements here can take a natural stacked layout in mobile viewports.